server/server.js

```diff
@@ -1,14 +1,25 @@
 const path = require('path');
 const express = require('express');
+const hentDekoratør = require('./dekoratør');
+const mustacheExpress = require('mustache-express');
```

Comment:
Mustache brukes til å erstatte NAV_SCRIPTS, NAV_STYLES osv. i HTML-filen. Mustache-express er for at express skal klare å gjøre dette med index-fila når den serves.